.vscode/settings.json

```diff
@@ -0,0 +1,14 @@
+{
+  "editor.formatOnSave": true,
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "eslint.packageManager": "yarn",
+  "eslint.validate": [
+    "javascript",
+    "javascriptreact",
+    "html",
+    "typescriptreact"
+  ],
+  "[shellscript]": {
+    "editor.defaultFormatter": "foxundermoon.shell-format"
+  }
```

Comment:
I don't think I'd include this since there aren't any bash scripts in the repo yet

Reply:
OK.  This is a copy and paste from the hypersync-sdk which also doesn't have any scripts, but I'll remove it here.

Reply:
Removed.